docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -99,17 +89,113 @@ Scanning is Semgrep's primary operation. When you first sign into Semgrep App, i
 
 Semgrep App enables users to choose what findings prevent a pull or merge request (PR or MR) from merging into the repository. Setting these blocking and non-blocking rules is achieved through the Rule Board.
 
-### Adding a project
+### Scanning a new project 
+
+A **project** is a repository from either:
 
-A **project** is a repository from your GitHub or GitLab account that you add to Semgrep App for scanning. Semgrep App can run scans on many projects with rules set in the Rule Board. When you add a project, Semgrep scans it for the first time with pre-selected rules chosen based on the repository's language and framework.
+* Your GitHub or GitLab account that you add to Semgrep App for scanning. Projects from GitHub or GitLab are integrated through Semgrep App.
+* A local git repository in your machine. Projects from your local machine are integrated through Semgrep CLI.
 
+Semgrep App can run scans on many projects with rules set in the Rule Board. First-time Semgrep App users scan projects with pre-selected rules chosen based on the repository's language and framework. To view these pre-selected rules, see the [Registry default ruleset](https://semgrep.dev/p/default).
 
-To add a project:
+Over time, users modify the Rule Board with rules specific to their codebase's security or business goals.  
+
+#### Option A: Adding a local repository through Semgrep CLI
+
+Adding a local repository is considered a standalone scan. This means the scan is manually triggered and is not continuous, unlike a CI job. 

```

Comment:
```suggestion
Adding a local repository enables standalone scans. This means scans are manually triggered and are not continuous, unlike scans run in a CI job. 

```

Reply:
Hmm... there's a subtle difference between "enabling a standalone scan" and the implication that a scan is treated by semgrep as standalone. Maybe the problem lies in "adding a local repository". There is no feature that is technically "enabled" here. I am thinking:

> Scanning a project from the CLI is a standalone action. This means that you manually triggered the scan. Scans from CLI are not continuous nor scheduled, unlike scans run in a CI job.

Does this work? @pabloest

Reply:
@s-santillan Yes, that’s much better than what I wrote :)